roles/migrationcontroller/templates/mig_host_cluster.yml.j2

```diff
@@ -7,5 +7,8 @@ metadata:
 spec:
 {% if azure_resource_group != '' %}
   azureResourceGroup: "{{ azure_resource_group }}"
+{% endif %}
+{% if host_cluster_restart_restic is defined %}
```

Comment:
@jmontleon does this look right? I only want to set this on the MigCluster CR if the user has defined the var `host_cluster_restart_restic ` on the MigrationController CR.

Reply:
Looks right to my eye. Will host_cluster_restic_restart only control restic restarts on the 'host' migcluster? What about the other cluster?

Or does this mean the one hte MigrationConttroller CR is installed on in this case (which I think is actually the case... 'host' just seems to be getting overloaded)

Reply:
Correct, `host_cluster_restic_restart` only controls this setting on the "host" cluster, we can change any other MigCluster directly. 

The issue I am trying to avoid is an overwrite of the "host" MigCluster `restartRestic` field if we told the user to edit this manually. In most cases where users are running MTC on 4.x, the source cluster will be the only one needing this setting, so I don't expect this operator var will often be used.

Reply:
Makes sense!